src/components/Controls.vue

```diff
@@ -35,6 +35,10 @@ export default {
       workspace: null,
       graph: null,
       selectNeighbors: true,
+      labelVariable: "_key",
+      colorVariable: "_key",
```

Comment:
Maybe we can leave these with a default value of `null`, and add an option to each dropdown called `(none)`, and use that to avoid placing a label or a special color at all. This might be more important for the color field, because while _key is possibly appropriate for a default label, it definitely is the wrong choice for a color.

Reply:
I think we should leave the label as _key since it's strange if no label renders.

For colorVariable, we could default to table which is most likely going to be the node type for any given data. Changing to null also seems to be an appropriate solution

Reply:
Oh, yeah, table is a good choice for the color option.

Reply:
Fixed in color = d3.scale.ordinal()
          .range(["#1f77b4", "#ff7f0e", "#2ca02c", "#d62728", "#9467bd", "#8c564b", "#e377c2"])
          .domain(d3.range(0,7));